File: angular/src/app/models/weather-info.ts
export class WeatherInfo {
    description: string;
    weatherState: string;
    temperature: number;
    maxTemperature: number;
    minTemperature: number;

    public WeatherInfo(temp: number, maxTemp: number, minTemp: number, state: string, description: string) {
        this.description = description;
        this.weatherState = state;
        this.temperature = temp;
        this.maxTemperature = maxTemp;
        this.minTemperature = minTemp;
    }
}

File: java/src/main/java/com/todos/api/model/WeatherInfo.java
package com.todos.api.model;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class WeatherInfo {
    int temperature;
    int maxTemperature;
    int minTemperature;
    String weatherState;
    String description;

    public WeatherInfo() {
    }

    public WeatherInfo(int temperature, int maxTemperature, int minTemperature, String weatherState, String description) {
        this.temperature = temperature;
        this.maxTemperature = maxTemperature;
        this.minTemperature = minTemperature;
        this.weatherState = weatherState;
        this.description = description;
    }

    @Override
    public String toString() {
        return "{" +
                "\"temperature\":" + "\"" + temperature + "\"," +
                "\"maxTemperature\":" + "\"" + maxTemperature + "\"," +
                "\"minTemperature\":" + "\"" + minTemperature + "\"," +
                "\"weatherState\":" + "\"" + weatherState + "\"," +
                "\"description\":" + "\"" + description + "\"" +
                '}';
    }
}
